src/components/views/messages/MImageReplyBody.js

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 Tulir Asokan <tulir@maunium.net>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+import { _td } from "../../../languageHandler";
+import * as sdk from "../../../index";
+import MImageBody from './MImageBody';
+import MFileBody from "./MFileBody";
+
+export default class MImageReplyBody extends MImageBody {
+    onClick(ev) {
+        ev.preventDefault();
+    }
+
+    wrapImage(contentUrl, children) {
+        return children;
+    }
+
+    // Don't show "Download this_file.png ..."
+    getFileBody() {
+        return MFileBody.prototype.presentableTextForFile.call(this, this.props.mxEvent.getContent());
```

Comment:
presentableTextForFile is an inherently static function, can we extract it somewhere saner to reuse?

Reply:
I moved it outside the `MFileBody` class and made it an exported function